oak_remote_attestation_noninteractive/proto/v1/messages.proto

```diff
@@ -24,10 +24,23 @@ option java_package = "oak.session.noninteractive.v1";
 message PublicKeyInfo {
   // The serialized enclave public key.
   // TODO(#3442): Specify key format.
-  bytes public_key = 1;
+  bytes enclave_public_key = 1;
   // The serialized attestation report binding the public key.
   bytes attestation = 2;
-  // TODO(#3442): Add evidence (endorsements and inclusion proofs) for the attested binary.
+  // The serialized AMD-SEV-SNP certificate.
+  bytes tee_certificate = 3;
+  // The serialized binary attestation evidence.
+  bytes binary_attestation = 4;
+  // The serialized server configuration. The details of the configuration is application-specific.
+  // This field is optional and can be left empty.
+  bytes server_config = 5;
```

Comment:
I assume this config must be bound to the attestation report in some way, so either this must be signed using the enclave_public_key (if it is a signing key), or its hash should apear in the attestation report's additional data. We should decide which of these approaches to follw and be clearer in the comment on how we prove the integrity of this field.

Reply:
Good point. The last thing I remember is that the config hash should be included in the attestation report's additional data. I'll update the comment accordingly.

Reply:
For now I think that is fine. If we add support in future for a signing key we can perhaps use that to sign the server config and any additional data from the TEE.